tools/sv-report

```diff
@@ -115,6 +115,8 @@ top_dir = os.path.abspath(os.curdir)
 logs_out_dir = os.path.join(out_dir, "logs")
 
 lex = lexers.get_lexer_by_name("systemverilog")
+formatter = HtmlFormatter(
```

Comment:
I think we can't use single ``formatter`` object, as it looks like ``HtmlFormatter`` changes its state when it formats file, e.g.: https://github.com/pygments/pygments/blob/3f4c7331a18b073209297a125131f860b4bd8220/pygments/formatters/html.py#L861